packages/vsce/src/commands/inspectResourceCommand.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { commands, ExtensionContext, window, workspace } from "vscode";
+import { ResourceInspectorViewProvider } from "../trees/ResourceInspectorViewProvider";
+import { getFocusRegion } from "./setFocusRegionCommand";
+import { IFocusRegion } from "./IFocusRegion";
+import { Resource, ResourceContainer } from "../resources";
+import { IResource, IResourceMeta } from "../doc";
+import { SupportedResources } from "../model";
+import { CICSLogger } from "../utils/CICSLogger";
+import { CICSMessages } from "../constants/CICS.messages";
+
+export function getInspectResourceCommand(context: ExtensionContext) {
+  return commands.registerCommand("cics-extension-for-zowe.inspectResource", async (resourceName: string, resourceType: string) => {
+    if (!await workspace.getConfiguration().get('zowe.cics.resourceInspector')) {
+      return;
+    };
+
+    const focusRegion: IFocusRegion = await getFocusRegion();
+
+    if (focusRegion) {
+      const type = getResourceType(resourceType);
+
+      if (!type) {
+        // Error if resource type not found
+        CICSLogger.error(CICSMessages.CICSResourceTypeNotFound.message);
+        window.showErrorMessage(CICSMessages.CICSResourceTypeNotFound.message);
+        return;
+      }
+
+      const resourceContainer = new ResourceContainer<IResource>(type);
+      resourceContainer.setCriteria([resourceName]);
```

Comment:
We don't check resourceName is actually a value. Not stop-ship for now but worth considering some validation in your follow-up.